documentation/src/main/asciidoc/public/reference/_integrations.adoc

```diff
@@ -25,3 +25,40 @@ Check the https://hibernate.org/search/releases/#compatibility-matrix[compatibil
 to find versions of Hibernate Search compatible with these older versions of Hibernate ORM.
 
 Use <<other-integrations-jakarta,Hibernate ORM 7>> instead.
+
+[[other-integrations-lucene-next]]
+== Lucene 10
+
+include::../components/_incubating-warning.adoc[]
+
+Hibernate Search {hibernateSearchVersionShort} provides two variations of the <<backend-lucene,Lucene backend>>.
```

Comment:
AFAIK a "variation" is always relative to something, so either you say "backend-lucene-next is a variation of backend-lucene" or "there are two variants".

```suggestion
Hibernate Search {hibernateSearchVersionShort} provides two variants of the <<backend-lucene,Lucene backend>>.
```